config/licenses.yml

```diff
@@ -82,7 +82,7 @@
   deprecated: false
   label: "PDDL-1.0 Public Domain Dedication and License"
   group: "Open Data Commons (ODC) licenses"
-- id: ""
+- id: "no-license"
```

Comment:
Using "" to indicate was no license was problematic because it conflicted with use of "" for a prompt. Opted to change this to stay on the rails.